cuda_core/cuda/core/experimental/_launcher.py

```diff
@@ -63,28 +65,45 @@ def _cast_to_3_tuple(self, cfg):
             raise ValueError
 
 
+# binding availability depends on cuda-python version
+py_major_minor = tuple(int(v) for v in (
+    importlib.metadata.version("cuda-python").split(".")[:2]))
+driver_ver = handle_return(cuda.cuDriverGetVersion())
```

Comment:
This would load libcuda and bind all driver symbols, which we defer as late as possible in the binding. Would be nice to do the same in core.

Reply:
Addressed in commit b634d700.